test/plots/traffic-horizon.js

```diff
@@ -0,0 +1,31 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/traffic.csv", d3.autoType);
+  const bands = 7;
+  const step = +(d3.max(data, d => d.value) / bands).toPrecision(2);
+  return Plot.plot({
+    facet: { data, y: "name" },
+    fy: { axis: null, domain: new Set(data.map((d) => d.name)) },
+    x: { axis: "top" },
+    y: { axis: null, domain: [0, step] },
+    color: { scheme: "blues", range: [0.05, 1], type: "ordinal", legend: true },
```

Comment:
Feels like a bit of a bummer than the range is needed here, like these color schemes don’t work out of the box? Is there something about this application that makes the ColorBrewer schemes to light, compared to say a choropleth?

Reply:
Just a tiny aesthetic preference I had in the notebook—we can remove it.

Reply:
I remember now why I played with this: the thing is that the first band represents values between 0 and 1 * step, and the paler shade of blue should represent 0; here what I'd really want is the color representing step/2. I'll make this more explicit.